wrangler.toml

```diff
@@ -1,4 +1,4 @@
-name = "your-plugin-name"
+name = "ubiquibot-issue-comment-embeddings"
```

Comment:
UbiquiBot term is deprecated in favor of UbiquityOS (or when lowercase, ubiquity-os)